README.md

```diff
@@ -1,27 +1,92 @@
+The core of the website is built in React, with documentation and other content built in MDX.
+
+### Setting up a Development Environment
+The easiest way to start working on the documentation is by using a DEM such as Daytona.
+You can create a Workspace with the dependencies and configuration needed to preview the docs locally by executing the following command:
+
+```shell
+daytona create https://github.com/daytonaio/docs.git
+```
+
+Alternatively, if your local machine has Node.js and Yarn set up, you can clone this repository and execute `yarn install` to install the required dependencies.
+
+### Useful Commands
+This repository uses Yarn to handle project management.
+For convenience, a number of scripts are defined in `package.json` that can be called using Yarn:
 
 | Command                    | Action                                           |
 | :------------------------- | :----------------------------------------------- |
-| `yarn install`             | Installs dependencies                            |
 | `yarn run dev`             | Starts local dev server at `localhost:4321`      |
 | `yarn run build`           | Builds your production site to `./dist/`         |
 | `yarn run preview`         | Previews your build locally before deployment    |
 | `yarn run astro ...`       | Runs CLI commands like `astro add`, `astro check`|
 | `yarn run astro -- --help` | Provides help for using the Astro CLI            |
 
-## 👀 Looking for more information on how to structure and develop docs with Astro and Starlight?
+### Navigating the Docs
+On the filesystem, you'll find the content in `./src/content/docs` in the form of `.mdx` files.
+Daytona's documentation is organized into broad topics:
+
+*   __Configuration__   
+    Information around configuring the main components of Daytona.
+*   __Explanation__  
+    Background and conceptual information about Daytona and DEMs.
+*   __Installation__  
+    Information around installing Daytona under different environments.
+*   __Reference__  
+    Technical and architectural information about Daytona.
+*   __Usage__  
+    Practical information on using Daytona.
+
+## Contributing
+Daytona is licensed under the [Elastic License 2.0](LICENSE). If you would like to contribute to the software, you must:
+
+1. Read the Developer Certificate of Origin Version 1.1 (https://developercertificate.org/)
+2. Sign all commits to the Daytona project.
+
+This ensures that users, distributors, and other contributors can rely on all the software related to Daytona being contributed under the terms of the [License](LICENSE). No contributions will be accepted without following this process.
+
+## License
+This repository contains the Daytona Enterprise Demo installer, covered under the [Apache License 2.0](LICENSE), except where noted (any Daytona logos or trademarks are not covered under the Apache License 2.0, and should be explicitly noted by a LICENSE file.)
```

Comment:
C/p from our other repo. Update to sync with Daytona Docs.

Reply:
Updated in [4560ea5](https://github.com/daytonaio/docs/pull/23/commits/4560ea576d9e99e8cdce0b6dec6054ca01e6bfaf).